### DISCUSSION TITLE
ShowMessageBox MessageBoxButtons.YesNo disabling the Esc key and Window "X" close functionality

### DISCUSSION THREAD

**Discussion by duners88** [canton7/Stylet#309] [Ideas]
It would be nice if the MessageBoxButtons.YesNo parameter option disabled the Window close "X" and the ESC key functionality that closes the window.  
I use the YesNo enum value when I want to force the user to actively click something.  
I use the YesNoCancel if the user does not need to make a decision and can "opt out" or close the message without making a decision.

Just a suggestion, I believe this is how the System.Windows.Forms.MessageBox class functions.

**Comment by canton7** [comment]
The aim is to mirror System.Windows.MessageBox -- if this implements that behaviour, it's reasonable that Stylet should too. Created #317
